data/scripts/bin/create-pinned-image-sets.sh.template

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+export KUBECONFIG="/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig"
+
+# If the pinned images file already exist then we don't need to do anything
+if [ -f /etc/crio/crio.conf.d/50-pinned-images ];
```

Comment:
Could be nicer to use ConditionPathExists in the service (e.g. see: https://github.com/openshift/appliance/blob/1ad4148429d8445f774fb5308d23b6dbead70c15/data/services/bootstrap/pre-install-node-zero.service#L5).
I.e. try adding to create-pinned-image-sets.service:
```
ConditionPathExists=!/etc/crio/crio.conf.d/50-pinned-images
```